packages/editor-ui/src/views/WorkflowsView.vue

```diff
@@ -299,8 +305,8 @@ const WorkflowsView = defineComponent({
 				source: 'Workflows list',
 			});
 		},
-		getTemplateRepositoryURL(category: string) {
-			return this.templatesStore.getWebsiteCategoryURL(category);
+		getTemplateRepositoryURL() {
+			return this.templatesStore.getWebsiteTemplateRepositoryURL;
```

Comment:
Is this a function? Having a `get` prefix makes it look like its callable

Reply:
It's a getter (so technically it's a function) but agree, `get` is superfluous here. Will remove.